nim/whip/config.yaml

```diff
@@ -0,0 +1,3 @@
+framework:
+  github: mattaylor/whip
+  version: 0.0
```

Comment:
is it not in `0.1` ?

Reply:
Nope :(

Reply:
https://github.com/mattaylor/whip/blob/master/whip.nimble#L3

Reply:
Done.